client/src/main/java/com/battleships/start_window/connection/ConnectInfo.java

```diff
@@ -0,0 +1,6 @@
+package com.battleships.start_window.connection;
+
+public class ConnectInfo {
```

Comment:
Encapsulation abandoned.
Constructor with ip and port.
Getters if needed.

Reply:
Done